src/Shared/USConverters/FundStatisticsConverter.ts

```diff
@@ -78,24 +80,33 @@ export class FundStatisticsConverter extends MorningstarConverter {
 
                 ++rowIndex;
             }
+            metadata.PortfolioAnalyzed = fundStatistics.PortfolioAnalyzed;
         }
 
         if (securityBreakdown) {
             for (const security of securityBreakdown) {
-                const securityItem = security.FundStatisticsItem;
+                const securityItem = security.FundStatisticsItem,
+                    securityId = security.SecurityId;
                 let rowIndex = 0;
 
                 // Set empty columns for all securities
-                table.setColumn('Value_' + security.SecurityId);
+                table.setColumn('Value_' + securityId);
+
+                // Set metadata props for each security
+                metadata[`Analyzed_${securityId}`] = security.Analyzed;
+                metadata[`NotAnalyzed_${securityId}`] = security.NotAnalyzed;
```

Comment:
Can we go even further with consts and do something like this?

`const { FundStatisticsItem, SecurityId, Analyzed, NotAnalyzed } = security;`